tests/AnnotationFinderTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+use Collective\Annotations\AnnotationFinder;
+use PHPUnit\Framework\TestCase;
+use Mockery as m;
+
+class AnnotationFinderTest extends TestCase
+{
+    /**
+     * @var m\LegacyMockInterface|m\MockInterface|Illuminate\Contracts\Foundation\Application
+     */
+    protected $app;
+
+    public function setUp(): void
+    {
+        $this->app = m::mock('Illuminate\Contracts\Foundation\Application');
+    }
+
+    public function tearDown(): void
+    {
+        m::close();
+    }
+
+    /**
+     * @dataProvider scannedChecksDataProvider
+     *
+     * @covers \Collective\Annotations\AnnotationFinder::routesAreScanned
+     * @covers \Collective\Annotations\AnnotationFinder::getScannedRoutesPath
+     * @covers \Collective\Annotations\AnnotationFinder::eventsAreScanned
+     * @covers \Collective\Annotations\AnnotationFinder::getScannedEventsPath
+     * @covers \Collective\Annotations\AnnotationFinder::modelsAreScanned
+     * @covers \Collective\Annotations\AnnotationFinder::getScannedModelsPath
+     */
+    public function testRoutesAreScannedReturnsTrue($filepath, $found, $method, $expected)
+    {
+        $this->app->shouldReceive('make')
+            ->with('files')->once()
+            ->andReturn($files = m::mock('Illuminate\Filesystem\Filesystem'));
```

Comment:
Do we have other behaviour to expect on `m::mock('Illuminate\Filesystem\Filesystem')`?

Reply:
What are you thinking of? The next code block sets that up.
Or is it confusing that it's assigned here? And then the assertions added? I'll move it around